web/repositories.bzl

```diff
@@ -468,10 +493,10 @@ def org_mozilla_firefox():
   platform_http_file(
       name="org_mozilla_firefox",
       amd64_sha256=
-      "6d338e98b1ac1078738dcdab11d6a0c855e5087fa0d7d99dbffa521d16a45327",
+      "9adf41f9c8ed525906d270c66ed2cfbd9e2154a3aeccaadbb0bdfb6ce3a3ca73",
       amd64_urls=[
-          "https://ftp.mozilla.org/pub/firefox/releases/53.0.3/linux-x86_64/en-US/firefox-53.0.3.tar.bz2",
-          "http://mirror.bazel.build/ftp.mozilla.org/pub/firefox/releases/53.0.3/linux-x86_64/en-US/firefox-53.0.3.tar.bz2",
+          "https://ftp.mozilla.org/pub/firefox/releases/56.0.1/linux-x86_64/en-US/firefox-56.0.1.tar.bz2",
+          "http://mirror.bazel.build/https://ftp.mozilla.org/pub/firefox/releases/56.0.1/linux-x86_64/en-US/firefox-56.0.1.tar.bz2",
```

Comment:
mirror.bazel.build goes first. also should be `http://mirror.bazel.build/ftp.mozilla.org/pub/firefox/releases/56.0.1/linux-x86_64/en-US/firefox-56.0.1.tar.bz2`

Reply:
Done.